core/src/test/java/com/linecorp/armeria/server/logging/structured/ServiceLogJsonFormatTest.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.logging.structured;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.apache.thrift.TApplicationException;
+import org.apache.thrift.protocol.TMessage;
+import org.apache.thrift.protocol.TMessageType;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.armeria.common.thrift.ApacheThriftCall;
+import com.linecorp.armeria.common.thrift.ApacheThriftReply;
+import com.linecorp.armeria.server.logging.structured.util.ServiceLogJsonFormat;
+import com.linecorp.armeria.service.test.thrift.main.HelloService;
+import com.linecorp.armeria.service.test.thrift.main.HelloService.hello_args;
+import com.linecorp.armeria.service.test.thrift.main.HelloService.hello_result;
+
+public class ServiceLogJsonFormatTest {
+    private static final ObjectMapper customObjectMapper = ServiceLogJsonFormat.objectMapper();
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private static final long TIMESTAMP_MS = 12345;
+    private static final long RESPONSE_TIME_MS = 6789;
+    private static final String THRIFT_SERVICE_NAME = HelloService.class.getCanonicalName();
+    private static final String THRIFT_METHOD_NAME = "hello";
+
+    private static ApacheThriftRpcServiceLogEntry getEntry(ApacheThriftCall call, ApacheThriftReply reply) {
+        return new ApacheThriftRpcServiceLogEntry(
+                TIMESTAMP_MS,
+                RESPONSE_TIME_MS,
+                THRIFT_SERVICE_NAME,
+                THRIFT_METHOD_NAME,
+                call, reply);
+    }
+
+    private static void compareJson(String got, String expected) throws IOException {
```

Comment:
with respect to which convention?
It's just a private method local variable. I don't think it's a big deal here but let me know if there's an obvious reason that I better follow that.